### DISCUSSION TITLE
How to get an older chrome os version / How to downgrade my chromebook?

### DISCUSSION THREAD

**Discussion by 88411** [3kh0/ext-remover#1185] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I am trying to downgrade my chrome book to v128 (I am on v129) since rigtools got patched this update. I don't really even know if my chromebook will let me. I know how to do it the only problem is chrome 100 is down so I don't know where to download the older version from. (PS. my board is dedede) If I can't downgrade I could use cryptosmite by physically opening the chromebook, however I don't really want to unenroll. I just want to remove go guardian.

Can you guys verify if it is possible to downgrade you chromebook with the chrome recovery util and thumbdrive?
Can you guys help me find a v128 chrome os for dedede? 
If it isn't possible to downgrade by default, is their anything I can do to my motherboard to allow it?(Like short 2 pins)
I know you can short two pins to get cryptosmite to work but will something like that allow me to downgrade?
Is there another way to use rigtools on v129?
My kernver is 4 btw

**Comment by SniperWasTaken** [comment, marked as answer]
Unfortunately, if your kernel version is 4, then that sadly means your Chromebook is locked down, and no unenrollment tool/method (except the [**!!RISKY OPINION!!**](https://blog.darkn.bio/blog/3-the-tsunami) which only supports TI50 boards such as octopus, dedede, et cetera, et cetera) in the future would support it.

So yeah, sorry man.

**Reply by 88411** [reply]
Really? I heard that you shouldn't do the risky method on ti50.(according to Darkn's blog) I have dedede but it is cr50. I can't even downgrade one version at all? 

When it comes to the risky method I know shorten those pins will allow me to unenrolled my Chromebook with cryptosmite however, will shorting those two pins in the risky method just allow me to downgrade instead of fully unenrolling.

Preferably my goal would be to downgrade to version 128 where I can use rigtools to remove go guardian while still staying enrolled.

**Reply by 88411** [reply]
im keyrolled so this donesn't work

**Reply by spacedude234** [reply]
ummmmmm? 

"4.1. if you have a keyrolled dedede then you can buy a [suzy qable](https://www.ebay.com/itm/335130747039)(highly recommend this seller, bought from them and they're highly reputable) and use alternative steps i can provide/walk you through"

**Reply by BopMooCow23** [reply]
or you can just use kvs? (it worked for me on kernel 4)

**Reply by prjamming** [reply]
Yes, you do

**Reply by ghost** [reply]
no u can unenroll using guest exploit for v130

**Reply by JackWagon885** [reply]
there's icarus & stuff nowadays

**Comment by schoolexploitkid** [comment]
You can downgrade to any version that is 125 or above 125 i think

**Comment by Theaveragebacon** [comment]
There is a very risky and magical method.
Requirements:
Chromebook insurance
A random person you blame/frame (If you fail)

Depending on the model all you have to do is power wash. Hold refresh and esc keys down. Then click the power button on your Chromebook (Usually on the right side.) Then click control d then pray to god this works. Click enter. Use a hotspot and connect to your school account and your home account. It should work fine. Also don't trust me. I'm a little gremlin hehehehe. (search up little gremlin and click images. I'm the 2nd row 4th image).

**Reply by Theaveragebacon** [reply]
If you fail blame the random person.
If you give me your Chromebook model I could find a actual bypass.

**Reply by givbd** [reply]
ok bro its not that risky 
also you could powerwash using shift ctrl alt r
